packages/cozy-client/src/hooks/useQuery.js

```diff
@@ -84,7 +84,15 @@ const useQuery = (queryDefinition, options) => {
     return client.query(generateFetchMoreQueryDefinition(queryState), { as })
   }, [as, client])
 
-  return { ...queryState, fetchMore: fetchMore }
+  const fetch = useCallback(() => {
+    return client.query(definition, options)
+  }, [client, definition, options])
+
+  return {
+    ...queryState,
+    fetchMore: fetchMore,
+    fetch: fetch
```

Comment:
Good change! Let's add a test to document this feature and ensure no regression 🙏

Reply:
added, thanks :)